features/base/participants/reducer.js

```diff
@@ -131,6 +133,18 @@ ReducerRegistry.register('features/base/participants', (state = [], action) => {
     case PARTICIPANT_LEFT:
         return state.filter(p => p.id !== action.participant.id);
 
+    /**
+     * Remove all participants when connection is disconnected.
+     */
+    case CONNECTION_DISCONNECTED:
+        return [];
```

Comment:
This was more a safe check to ensure that all participants are cleared after CONNECTION_DISCONNECTED. 
But I agree that clearing only local tracks here is more clear.